src/pages/ProductsPage/ProductCard/styles.ts

```diff
@@ -0,0 +1,81 @@
+import { SxProps } from '@mui/material';
+
+export const mainCardStyle: SxProps = {
+  maxWidth: 400,
+};
+
+export const cardMediaContainer: SxProps = {
+  position: 'relative',
+};
+
+export const cardMediaChildContainer: SxProps = {
+  position: 'absolute',
```

Comment:
what is the reason use absolute here?

Reply:
I use 'absolute' for placing text under the image